server/src/main/java/org/eclipse/openvsx/repositories/RepositoryService.java

```diff
@@ -260,27 +260,27 @@ public PersonalAccessToken findAccessToken(long id) {
         return persistedLogRepo.findByTimestampAfterOrderByTimestampAsc(dateTime);
     }
 
-    public Streamable<ExtensionDTO> findAllActiveExtensionDTOsByPublicId(Collection<String> publicIds) {
+    public List<ExtensionDTO> findAllActiveExtensionDTOsByPublicId(Collection<String> publicIds) {
         return extensionDTORepo.findAllActiveByPublicId(publicIds);
     }
 
     public ExtensionDTO findActiveExtensionDTOByNameAndNamespaceName(String name, String namespaceName) {
         return extensionDTORepo.findActiveByNameIgnoreCaseAndNamespaceNameIgnoreCase(name, namespaceName);
     }
 
-    public Streamable<ExtensionDTO> findAllActiveExtensionDTOsById(Collection<Long> ids) {
+    public List<ExtensionDTO> findAllActiveExtensionDTOsById(Collection<Long> ids) {
         return extensionDTORepo.findAllActiveById(ids);
     }
 
-    public Streamable<ExtensionVersionDTO> findAllActiveExtensionVersionDTOsByExtensionId(Collection<Long> extensionIds) {
+    public List<ExtensionVersionDTO> findAllActiveExtensionVersionDTOsByExtensionId(Collection<Long> extensionIds) {
         return extensionVersionDTORepo.findAllActiveByExtensionId(extensionIds);
     }
 
-    public Streamable<FileResourceDTO> findAllFileResourceDTOsByExtensionVersionIdAndType(Collection<Long> extensionVersionIds, Collection<String> types) {
+    public List<FileResourceDTO> findAllFileResourceDTOsByExtensionVersionIdAndType(Collection<Long> extensionVersionIds, Collection<String> types) {
         return fileResourceDTORepo.findAllByExtensionIdAndType(extensionVersionIds, types);
     }
 
-    public Streamable<ExtensionReviewCountDTO> countAllActiveReviewsByExtensionId(Collection<Long> extensionIds) {
+    public List<ExtensionReviewCountDTO> countAllActiveReviewsByExtensionId(Collection<Long> extensionIds) {
```

Comment:
Shouldn't a "count" method return a number?

Reply:
The method returns the review count for each extension in bulk. `ExtensionReviewCountDTO` is a tuple of `extensionId` and `reviewCount`. What would be a better name for this method?